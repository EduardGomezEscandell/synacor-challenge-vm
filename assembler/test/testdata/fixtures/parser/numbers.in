0
16
0x15
0b011100
00