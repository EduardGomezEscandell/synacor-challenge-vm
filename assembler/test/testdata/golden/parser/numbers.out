<Start>
  <P>
    <D>
      <NUMBER 0>
      <D>
    <EOL>
    <P>
      <D>
        <NUMBER 16>
        <D>
      <EOL>
      <P>
        <D>
          <NUMBER 21>
          <D>
        <EOL>
        <P>
          <D>
            <NUMBER 28>
            <D>
          <EOL>
          <P>
            <D>
              <NUMBER 0>
              <D>
            <EOL>
            <P>
              <END>
  <END>
